Hunting Queries/Microsoft 365 Defender/Email Queries/Email-RemediationAction-List.yaml

```diff
@@ -0,0 +1,27 @@
+id: xxxxxxxxxxxxxxxxxxxxxxxx
+name: Listing Email Remediation Actions via Explorer
+description: |
+  - Track each cases with Network Message ID
+  - Sort the users who got a number of actions 
+  - e.g. Soft Delete, Hard Delete, Move to junk folder, Move to deleted items 
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - EmailEvents
+tactics:
+- N/A
```

Comment:
It should be empty array instead of N/A, please correct this also.
<img width="104" alt="image" src="https://github.com/Azure/Azure-Sentinel/assets/104008048/b2751f44-fea3-429c-a3a5-42bc8f7dc732">